layers/meta-balena-raspberrypi/recipes-kernel/picontrol/picontrol.bb

```diff
@@ -10,8 +10,7 @@ SRC_URI = " \
 	file://0002-Search-config-file-in-mnt-boot.patch \
 "
 
-# Corresponds to tag raspberrypi-kernel_9.20200616-4.19.95+revpi1
-SRCREV ="72a58797b2eed6edab99795476108cfeec95c8d9"
+SRCREV ="raspberrypi-kernel_1%9.20220728-5.10.120+revpi1"
```

Comment:
Not sure if this is correct. We see a picontrol error and this may be the reason?

Reply:
The fetching is fixed now but I get some compilation errors when I compile PiControl for 64 bit systems. This can be part of another PR if the PiControl is needed.

Reply:
picontrol is needed, we cannot remove it

Reply:
Create an issue for PiControl compilation at https://github.com/RevolutionPi/piControl/issues/81